### ISSUE TITLE
Flag to use to avoid installing libtspi-dev.

### ISSUE THREAD

**Issue by lihanshang** [google/go-attestation#202]
Hey Guys
I am running a test and I am seeing 
```
# github.com/google/go-tspi/tspi
/go/pkg/mod/github.com/google/go-tspi@v0.2.1-0.20190423175329-115dea689aad/tspi/context.go:17:27: fatal error: trousers/tss.h: No such file or directory
 // #include <trousers/tss.h>
                           ^
compilation terminated.
```
I am assuming this is the related issue of not installing the libtspi-dev since I am running it on centos. May I know if I dont want TPM12 support what I should run to build? 

I check the relate issues here https://github.com/google/go-attestation/issues/163 but not pretty clear how I should use any flag to make it build.

Thanks!

**Comment by ericchiang** [comment]
You should be able to build without cgo and not include it:

```
CGO_ENABLED=0 go test
```

Does that work?

**Comment by tracefinder** [comment]
Hi @ericchiang,

it works. However, the problem arises when there is another library, which uses cgo. For example, https://github.com/miekg/pkcs11. In this case there is no way to turn off libtspi-dev dependency.

**Comment by ericchiang** [comment]
Ah I see. If you'd like to send a PR to add a build tag (something like "notspi"), we'd be happy to reivew.

**Comment by ericchiang** [comment]
Closed by #232
